addons/packages/secretgen-controller/vendir.yml

```diff
@@ -0,0 +1,13 @@
+apiVersion: vendir.k14s.io/v1alpha1
+kind: Config
+minimumRequiredVersion: 0.12.0
+directories:
+  - path: 0.10.3
+    contents:
+      - path: .
+        githubRelease:
+          slug: vmware-tanzu/carvel-secretgen-controller
+          tag: v0.10.3
+          assetNames: ["package.yml"]
+          checksums:
+            package.yml: 0fdcfbaff66b5bbfea0cef9b261c60552417ee008c5cb852e7604e5ea9215605
```

Comment:
btw since we have checksums within release notes they should be automatically picked up.

Reply:
should... but it doesn't seem to work for me.

```
vendir sync
Fetching: 0.10.3 + . (github release vmware-tanzu/carvel-secretgen-controller@v0.10.3)

vendir: Error: Syncing directory '0.10.3':
  Syncing directory '.' with github release contents:
    Finding checksums in release notes:
      Expected to find sha256 checksum for file 'package.yml'
```